content/pl/build-release-run.md

```diff
@@ -0,0 +1,18 @@
+## V. Buduj, wypuszczaj, uruchamiaj
+### Precyzyjnie oddziel etap budowania od uruchamiania
+
+[Codebase](./codebase) jest przetwarzany w (nie developerskie) wdrożenie w trzech etapach
+
+* Podczas *etapu budowania* kod z repozytorium jest konwertowany do wykonywalnej paczki tzw. *buildu*. Używając wersji kodu zdefiniowanej przez commit w procesie deploymentu, w etapie budowania aplikacji pobiera się, dołącza do projektu [zależności](./dependencies) i kompiluje niezbędne zasoby.
+* W *etapie wypuszczania* aplikacji brany jest build stworzony w poprzednim etapie i konfigurowany na bazie [ustawień](./config) obecnego wdrożenia.  Stworzony w ten sposób *release* zawiera zbudowane źródło kodu, jego konfigurację i jest gotowe do uruchomienia w wybranym środowisku.
```

Comment:
1. `s/W etapie wypuszczania/Podczas etapu publikacji`
2. `brany jest build` -> bardzo kolokwialnie to brzmi, spróbuj "używany" 
3. `Stworzony` jest release, ale potem piszesz w formie `jest gotowe`.